lib/reverse_sentence.rb

```diff
@@ -1,6 +1,31 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: o(n)^3
+# Space complexity: o(n)^2
+
 def reverse_sentence(my_sentence)
```

Comment:
This works, but there is an O(n) both space and time complexity.  These aren't polynomial in time/space complexity.